aws-lambda-java-runtime-interface-client/src/main/jni/Dockerfile.glibc

```diff
@@ -1,10 +1,11 @@
-# we use centos 6 to build against glibc 2.12
-FROM centos:6
+# we use centos 7 to build against glibc 2.12
```

Comment:
With CentOS 7 as a base, we are now building against `glibc` 2.17 - as a result, this comment and requirements in `README.md` should be updated to reflect this too.

Building against `glibc` 2.17 effectively means that the minimum supported version is 2.17 now